src/assets/css/style.css

```diff
@@ -13,7 +13,7 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    padding-top: 50%;
+    
```

Comment:
styling should be done through sx - I'd say only the background style is appropriate to have in a css file

Reply:
Moved the styling to the sx minus the background for this file.